### PR TITLE
Show successful investment on project pg

### PR DESCRIPTION
**Pull request by tsyaeger**
Removes equity field
Updates donor count, funding total, and progress bar with successful update.

### CODE REVIEW

File: client/src/components/payments/stripeStyle.js [hunk 1]
```diff
@@ -2,6 +2,68 @@
...
+            boxShadow: '0 16px 70px -12.125px rgba(0,0,0,0.3)'
+        },
+        [muiBaseTheme.breakpoints.down('sm')]: {
+            width: '100%',
+            padding: "0"
+        }
+    },
+    content: {
+        textAlign: 'center',
+        padding: muiBaseTheme.spacing.unit
```
Comment (evankp) [inline client/src/components/payments/stripeStyle.js]: just as an fyi, I think material-ui is deprecating this or something. got a warning on console about it on other pages

File: client/src/components/payments/stripeStyle.js [hunk 2]
```diff
@@ -2,6 +2,68 @@
 //of payment card. If you figure out how to use MUI
 //components for these items please let me know. -taryn
 
+const checkoutStyles = (muiBaseTheme) => ({
+    card: {
+        width: '500px',
+        margin: '50px auto',
+        padding: muiBaseTheme.spacing.unit * 3,
```
Comment (evankp) [inline client/src/components/payments/stripeStyle.js]: Same as below

**Comment by evankp** [comment]
But you do need to merge from dev I see ;)

File: client/src/components/payments/stripeStyle.js (hunk 1 above)
Comment (tsyaeger) [reply client/src/components/payments/stripeStyle.js]: yeah I saw that. what have you been using instead?

**Pull request merged**